dcp-0013/dcp-0013.mediawiki

```diff
@@ -0,0 +1,328 @@
+<pre>
+DCP: 0013
+Title: New Max Treasury Expenditure Policy
+Author: Dave Collins <davec@decred.org>
+Status: Active
+Created: 2025-11-20
+License: CC0-1.0
+License-Code: ISC
+Requires: DCP0006, DCP0007
```

Comment:
Does 13 actually require 7?

e.g. on a new testnet, I think 6 and 13 would operate just fine without 7.